src/GithubActionsToolHelper.ts

```diff
@@ -0,0 +1,75 @@
+import * as core from '@actions/core';
+import * as io from '@actions/io';
+import * as exec from '@actions/exec';
+
+export class GithubActionsToolHelper {
+
+    public getGithubRunId(): string {
+        return process.env['GITHUB_RUN_ID'] || '';
+    }
+
+    public getGithubRunNumber(): string {
+        return process.env['GITHUB_RUN_NUMBER'] || '';
+    }
+
+    public info(message: string): void {
+        core.info(message);
+    }
+
+    public error(message: string): void {
+        core.error(message);
+    }
+
+    public warning(message: string): void {
+        core.warning(message);
+    }
+
+    public debug(message: string): void {
+        core.debug(message);
+    }
+
+    public async exec(commandLine: string, args?: string[], execOptions?: exec.ExecOptions): Promise<number> {
```

Comment:
In that case, this method should return a Promise object similar to [this](https://github.com/Azure/container-apps-deploy-action/blob/snehapar/container-apps-deploy-action-javascript/src/Utility.ts#L13) to make sure we can access the error code, stdout and stderr. This is a customization we had to do to make sure the output and error steams can be accessed as well because the `exec` method only returns the error code.